apps/web/middleware.ts

```diff
@@ -76,15 +75,34 @@ const middleware: NextMiddleware = async (req) => {
   });
 };
 
+const routingForms = {
+  handle: (url: URL) => {
+    // Next.config.js Redirects don't handle Client Side navigations and we need that.
+    // So, we add the rewrite here instead.
+    if (url.pathname.startsWith("/routing-forms/")) {
+      url.pathname = url.pathname.replace(/^\/routing-forms\//, "/apps/routing-forms/");
+      return NextResponse.rewrite(url);
+    }
+
+    // Don't 404 old routing_forms links
+    if (url.pathname.startsWith("/apps/routing_forms")) {
+      url.pathname = url.pathname.replace(/^\/apps\/routing_forms\//, "/apps/routing-forms/");
+      return NextResponse.rewrite(url);
+    }
+  },
+  // Matcher paths that are required by the handle function to work
+  paths: ["/apps/routing_forms/:path*", "/routing-forms/:path*"],
```

Comment:
There shouldn't be many requests to these paths.
The path that would get the most traffic is the actual Routing Form publically available at /forms